winui/Masked-TextBox/Error-Indication.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: Error Indication in WinUI MaskedTextBox control | Syncfusion
+description: Explore the various error types supported by the Syncfusion WinUI MaskedTextBox (SfMaskedTextBox) control to enhance user experience and input validation.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# Error Indication in WinUI MaskedTextBox
+
+The [WinUI MaskedTextBox](https://www.syncfusion.com/winui-controls/masked-textbox) control provides comprehensive error types support to facilitate effective input validation and enhance user experience. This feature enables users to utilize various error icons and styles based on specific validation scenarios.The following error types such as none, default, warning, information, critical, success, custom is available in the MaskedTextBox control.
+
```

Comment:
@GilbertmanojSF3971 
# Error Indication in WinUI MaskedTextBox

The MaskedTextBox control supports error indication by displaying an icon and providing additional details upon hovering over the icon. This powerful feature allows users to employ various error icons based on specific validation scenarios.

## Built-in Error Types:

The MaskedTextBox control offers several built-in error types:

None: No visible signals for errors.
Default: Red bottom border brush in the MaskedTextBox Control.
Warning: Visual highlight for potential concerns needing immediate attention.
Information: Share non-essential details or guidance.
Critical: Urgent issues requiring immediate action.
Success: Positive confirmation for valid input or successful actions.

Reply:
@Manivannan-E Sir. Modified the description for Error Indication in WinUI MaskedTextBox Title.